.github/workflows/release_pod.yml

```diff
@@ -0,0 +1,66 @@
+name: Build and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      versionBumpLevel:
+        description: 'Version bump level (patch, minor, major)'
+        required: true
+        type: choice
+        default: 'patch'
+        options:
+        - patch
+        - minor
+        - major
+
+jobs:
+  build-and-release:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      
+      - name: Set up XCode 
+        run: sudo xcode-select --switch /Applications/Xcode_15.0.app
```

Comment:
We currently support MABS 9 (which uses Xcode 14.1), so we should select that version to build the `xcframework`.
Also, for `xcodebuild` to work, shouldn't we switch to `/Applications/Xcode_x.app/Contents/Developer`?

Reply:
From my testing this wasn't needed

Reply:
You're right. Just tested and even if we don't mention the `/Contents/Developer`, it will use it automatically. TIL